index.html

```diff
@@ -18,7 +18,11 @@
   <script>
     (function(window, videojs) {
       var player = window.player = videojs('videojs-vr-player');
-      player.vr({projection: '360', debug: true});
+      player.mediainfo = player.mediainfo || {};
```

Comment:
We may be able to read spacial metadata from the video file. I am not sure that would work for HLS or if we even have access to it for mp4